app/assets/stylesheets/era_custom.scss

```diff
@@ -16,3 +22,147 @@ div.terms-of-use-custom {
   margin-left: 30px;
   margin-right: 30px;
 }
+
+.dark {
+  background-color: #2d353d;
+  color: #fff;
+}
+
+h1,
+h2,
+h3,
+h4 {
+  font-family: 'Playfair Display', serif;
+}
+
+.footer {
+  width: 100%;
+  text-align: center;
+  font-size: 80%;
+}
+.footer h2 {
+  font-size: 150%;
+}
+.footer ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+.footer ul > li {
+  margin: 0;
+  padding: 0;
+}
+.footer ul > li > a {
+  font-size: 110%;
+  padding-bottom: 10px;
+}
+.footer a {
+  color: $link1 !important;
+}
+.footer a:hover {
+  color: $link1Hover;
+}
+.footer a:visited {
+  color: $link1Visited;
+}
+.footer a:hover:visited {
+  color: $link1Hover;
+}
+.footer .copyright {
+  padding: 40px 20px 0 20px;
+  font-size: 90%;
+}
+
+#masthead {
```

Comment:
Again, i'd move this into a `header.scss` file.

Also ID's should almost never be used in CSS or JS. You already have a `custom-header` class to work with from the html, Could change all these ID references to use the `custom-header` class and nothing would change.

Reply:
Also I'd take a look at how bootstrap does it's headers/navbar. Instead of writing all this custom CSS to do something similar.